demos/applications/icm20948.cpp

```diff
@@ -13,10 +13,25 @@
 // limitations under the License.
 
 #include "../hardware_map.hpp"
+#include <cmath>
 #include <libhal-icm/icm20948.hpp>
 #include <libhal-util/serial.hpp>
 #include <libhal-util/steady_clock.hpp>
 
+#define M_PI 3.14159265358979323846
+
+float computeHeading(float x, float y, float offset = 0.0)
```

Comment:
```suggestion
float compute_heading(float x, float y, float offset = 0.0)
```

The clang-tidy linter should catch this though. It may not be checking the demos folder though.